khmer/khmer_args.py

```diff
@@ -110,6 +263,15 @@ def __call__(self, parser, namespace, values, option_string=None):
 ** Your values for ksize, n_tables, and tablesize
 ** will be ignored.'''.format(hashfile=values))
 
+            if getattr(namespace, 'unique_kmers') != 0 or \
+               getattr(namespace, 'max_memory_usage') is not None or \
+               getattr(namespace, 'max_tablesize') is not None:
+
+                print("\n*** WARNING: you have specified a number of unique"
```

Comment:
How about:

you have asked that the graph size be calculated automatically (-U or -M), 
but are loading an existing graph. Size will NOT be set automatically.